components/org.wso2.carbon.identity.application.authenticator.samlsso/src/main/java/org/wso2/carbon/identity/application/authenticator/samlsso/logout/validators/LogoutReqSignatureValidator.java

```diff
@@ -94,12 +94,10 @@ public boolean validateSignature(String queryString, String issuer, X509Certific
      *
      * @param request SAML Assertion (SAML LogoutRequest).
      * @param cred    Signature signing credential.
-     * @param alias   Certificate alias against which the signature is validated.
      * @return true   If the signature is valid.
      * @throws IdentityException If signature validation process fails.
      */
-    public boolean validateXMLSignature(SignableXMLObject request, X509Credential cred,
-                                        String alias) throws IdentityException {
+    public boolean validateXMLSignature(SignableXMLObject request, X509Credential cred) throws IdentityException {
```

Comment:
As a practice it is not recommended to change the method signature of public methods. Therefore let's introduce a separate  method and deprecate this

Reply:
revert this change in this PR and will address this in another PR